frontend/README.md

```diff
@@ -173,9 +173,18 @@ stack exec cardano-keygen -- --dump-dev-genesis-keys keys/{}.key
 
 *2. Run `cardano-sl-explorer` (in another terminal window)*
 
+If you have `cardano-sl-explorer` and `cardano-sl` in the same folder:
+
+```bash
+cd {path/to/}cardano-sl-explorer
```

Comment:
@ksaric Add information to run `WALLET_TEST=1` before or add this to `./full-launch.sh`. Also make a note that `cardano-sl` has not been started separately anymore. Or even better, remove `Run cardano-sl` from `README`